CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 
 - Implemented CPU name grouping mechanism for analytics ([PR 3089](https://github.com/input-output-hk/daedalus/pull/3089))
 - Implemented rounding RAM size in analytics ([PR 3088](https://github.com/input-output-hk/daedalus/pull/3088))
+- Added custom dimension to track legacy and hardware wallets ([PR 3091](https://github.com/input-output-hk/daedalus/pull/3091))
```

Comment:
@marcin-mazurek could you move the log to the very top please 🙏

Reply:
Done